regen/HeaderParser.pm

```diff
@@ -1,6 +1,7 @@
 package HeaderParser;
 use strict;
 use warnings;
+BEGIN { require v5.9.5; }
```

Comment:
Why not just `use v5.10;` since that's the stable version that introduced named captures?

Reply:
I agree with @ilmari.